app/DoctrineMigrations/Version20181023135851.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
```

Comment:
Slett den gamle DB migration. Er kanskje lurt å slette begge to så du får en helt fresh en generert